### PR TITLE
Ben/docu 882

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/command-reference.md [hunk 1]
```diff
@@ -1316,28 +1328,157 @@ OPTIONS
...
+
+        --ssh-key  file
+                ssh key to use for authentication on remote host to perform automatic deployment
+                [Optional, default = <nothing>]
+
+        --ssh-port  int
+                ssh port to use to perform automatic deployment
+                [Optional, default = 22]
+
+        --use-sudo      use sudo for privileged commands while performing remote installation
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1356]: Too many spaces?

File: docs/command-reference.md [hunk 2]
```diff
@@ -1316,28 +1328,157 @@ OPTIONS
...
+                [Optional, default = false]
+
+        --ignore-host-checking  ignore strict host key checking for unknown hosts
+                [Optional, default = false]
 ```
 
 #### Mandatory Parameters
 
+:::info
+The Azure hive agent requires a ADLS Gen2 storage account and container name, this is only for the purposes of generating the correct location for the database. The container will not be accessed by the Hive agent and no data will be written to the container.
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1366]: Split this up into two sentences (full stop after 'name'). Also, the second 'Hive agent' should be lower case h.

File: docs/command-reference.md (hunk 1 above)
Comment (benradstone) [reply docs/command-reference.md:1356]: That's the output from the CLI, nothing to do with me!

File: docs/command-reference.md [hunk 3]
```diff
@@ -1316,28 +1328,157 @@ OPTIONS
...
+:::
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1367]: What's happening here? Is that a note or similar?

File: docs/command-reference.md [hunk 4]
```diff
@@ -1316,28 +1328,157 @@ OPTIONS
...
+
+Once all pre-requirements are met, see the [system-assigned identity](#system-assigned-identity) or [user-assigned identity](#user-assigned-identity) parameters.
+
+###### System-assigned identity
+
+No other parameters are required for a system-managed identity.
+
+###### User-assigned identity
+
+The `--client-id` parameter must be specified:
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1419]: Specify the client-id parameter (make it active)

File: docs/command-reference.md (hunk 3 above)
Comment (benradstone) [reply docs/command-reference.md:1367]: I don't know what you mean by this? Can you explain?

**Review by shonaisbister-disco — CHANGES_REQUESTED** [review]
Little nit-picky things. But we'll have to sort out the 'environment setup' section - likely a section in another topic?

File: docs/command-reference.md (hunk 3 above)
Comment (shonaisbister-disco) [reply docs/command-reference.md:1367]: I was wondering if the ::: on line 1367 was the start of note admonition formatting.

Comment (benradstone) [reply docs/command-reference.md:1367]: That's the end of the admonition:

```
:::note
enter admonition text here
:::
```

I've just used "info" as it's more informational than note-worthy.

Comment (shonaisbister-disco) [reply docs/command-reference.md:1367]: Oh dear! Sorry about that.

**Pull request merged**